config.go

```diff
@@ -104,6 +104,7 @@ type ConfigNetwork struct {
 	durationControlPDU   time.Duration
 	linkbpsControl       int64
 	linkbpsData          int64
+	cmdWindowSz          int   // cmd Window size in number of chunks.
 	netdurationDataChunk time.Duration
```

Comment:
That makes sense to have model specific configs in a separate file. I think you wouldn't even need a separate source file, you can include the BaseConfig object in the model and the model can add new configs.

But for this command window size, do you want it to be model b specific? I think command window size should be a global config option.